docs/guide/service/nlb.md

```diff
@@ -182,6 +182,8 @@ The controller automatically selects the worker node security groups that it mod
 
     `${cluster-name}` is the name of the Kubernetes cluster.
 
+In the case that you have more than one matching security group with the tag `kubernetes.io/cluster/${cluster-name}`, you may specify additional tags with the `endpoint-security-group-tags` to further specify the security group that should be used.
```

Comment:
By my reading of the code, it does not further refine only in the case where there is more than one matching security group, it requires all security groups to have the additional tags in order to be considered.

Reply:
Correct. I worded it this way because there should only be one matching security group in the end, but maybe this is better wording?

```
To further refine which security groups should be considered, use the `--service-target-eni-security-group-tags` flag. The controller will then only consider security groups that contain all the tags defined in this flag.
``` 

Feel free to suggest something better though :)